csvcubed-models/csvcubedmodels/rdf/namespaces.py

```diff
@@ -20,6 +20,8 @@
 
 RDFS = Namespace("http://www.w3.org/2000/01/rdf-schema#")
 
+CSVW = Namespace("http://www.w3.org/ns/csvw#")
```

Comment:
This change comes from your existing PR. We'll need to sort this one out after you've merged the other one.